File: fix.py
with open('c:\\Users\\kofiw\\Desktop\\cove\\src\\screens\\AddTransactionPanel.jsx', 'r', encoding='utf-8') as f:
    content = f.read()

content = content.replace('“', '"').replace('”', '"').replace('…', '...')

with open('c:\\Users\\kofiw\\Desktop\\cove\\src\\screens\\AddTransactionPanel.jsx', 'w', encoding='utf-8') as f:
    f.write(content)

File: fix.js
import fs from 'fs';
let content = fs.readFileSync('c:\\Users\\kofiw\\Desktop\\cove\\src\\screens\\AddTransactionPanel.jsx', 'utf8');
content = content.replace(/\u201C/g, '"').replace(/\u201D/g, '"').replace(/\u2026/g, '...');
fs.writeFileSync('c:\\Users\\kofiw\\Desktop\\cove\\src\\screens\\AddTransactionPanel.jsx', content);